mailbox/store/src/test/resources/eml/pgpSignedMail.json

```diff
@@ -10,121 +10,136 @@
-	],
-	"x-rc-virus": [
-	  "2007-09-13_01"
-	],
-	"x-loop": [
-	  "debian-security-announce@lists.debian.org"
-	],
-	"priority": [
-	  "urgent"
-	],
-	"x-virus-scanned": [
-	  "by amavisd-new using ClamAV at minet.net"
-	],
-	"x-spam-status": [
-	  "No, score=-1.51 required=1 tests=[BAYES_00=-1.5,\tT_RP_MATCHES_RCVD=-0.01] autolearn=unavailable"
-	],
-	"delivered-to": [
-	  "lists-debian-security-announce@bendel.debian.org"
-	],
-	"x-amavis-spam-status": [
-	  "No, score=-8.99 tagged_above=-10000 required=5.3\ttests=[BAYES_00=-2, DIGITS_LETTERS=1, FVGT_m_MULTI_ODD=0.02,\tLDO_WHITELIST=-5, OUR_MTA_MSGID=2, PGPSIGNATURE=-5,\tT_RP_MATCHES_RCVD=-0.01] autolearn=ham"
-	],
-	"resent-sender": [
-	  "debian-security-announce-request@lists.debian.org"
-	],
-	"x-original-to": [
-	  "lists-debian-security-announce@bendel.debian.org"
-	],
-	"to": [
-	  "debian-security-announce@lists.debian.org"
-	],
-	"list-id": [
-	  "<debian-security-announce.lists.debian.org>"
-	],
-	"old-return-path": [
-	  "<carnil@master.debian.org>"
-	]
-  },
+  "headers": [{
+    "name": "return-path",
+    "value": "<bounce-debian-security-announce=benwa=minet.net@lists.debian.org>"
+  }, {
+    "name": "received",
+    "value": "from mx2.minet.net (mx2.minet.net [192.168.102.26])\t by imap (Cyrus v2.4.16-Debian-2.4.16-4+deb7u1) with LMTPA;\t Wed, 03 Jun 2015 21:19:59 +0200"
+  }, {
+    "name": "x-sieve",
+    "value": "CMU Sieve 2.4"
+  }, {
+    "name": "received",
+    "value": "from localhost (spam.minet.net [192.168.102.97])\tby mx2.minet.net (Postfix) with ESMTP id 8A9DAA27DA1\tfor <benwa@minet.net>; Wed,  3 Jun 2015 21:20:03 +0200 (CEST)"
+  }, {
+    "name": "x-virus-scanned",
+    "value": "by amavisd-new using ClamAV at minet.net"
+  }, {
+    "name": "x-spam-flag",
+    "value": "NO"
+  }, {
+    "name": "x-spam-score",
+    "value": "-1.51"
+  }, {
+    "name": "x-spam-level",
+    "value": ""
+  }, {
+    "name": "x-spam-status",
+    "value": "No, score=-1.51 required=1 tests=[BAYES_00=-1.5,\tT_RP_MATCHES_RCVD=-0.01] autolearn=unavailable"
+  }, {
+    "name": "received",
+    "value": "from mx2.minet.net ([IPv6:::ffff:192.168.102.26])\tby localhost (spam.minet.net [::ffff:192.168.102.97]) (amavisd-new, port 10024)\twith ESMTP id cF6J0AAnzsTX for <benwa@minet.net>;\tWed,  3 Jun 2015 19:20:02 +0000 (UTC)"
+  }, {
+    "name": "received-spf",
+    "value": "None (no SPF record) identity=mailfrom; client-ip=82.195.75.100; helo=bendel.debian.org; envelope-from=bounce-debian-security-announce=benwa=minet.net@lists.debian.org; receiver=benwa@minet.net"
+  }, {
+    "name": "received",
+    "value": "from bendel.debian.org (bendel.debian.org [82.195.75.100])\t(using TLSv1.2 with cipher AECDH-AES256-SHA (256/256 bits))\t(No client certificate requested)\tby mx2.minet.net (Postfix) with ESMTPS id 7F69FA27D98\tfor <benwa@minet.net>; Wed,  3 Jun 2015 21:20:01 +0200 (CEST)"
+  }, {
+    "name": "received",
```

Comment:
it's not an issue to have several headers with the same name?

Reply:
No